doc/modules/array_api.rst

```diff
@@ -24,6 +29,12 @@ explicitly as explained in the following.
     Currently, only `array-api-strict`, `cupy`, and `PyTorch` are known to work
     with scikit-learn's estimators.
 
+The following video provides an overview of the standard's design principles
+and how it facilitates interoperability between array libraries:
+
+- `Scikit-learn on GPUs with Array API <https://www.youtube.com/watch?v=c_s8tr1AizA>`_
+  by `Thomas J. Fan`_ at PyData NYC 2023.
+
```

Comment:
I hope this YouTube video can help end-users when using the Array API. If not, I can have it removed.